contract/src/pegasus.js

```diff
@@ -306,26 +231,14 @@ const makePegasus = (zcf, board) => {
    */
   const connectionToLocalDenomState = makeWeakStore('Connection');
 
-  /**
-   * @type {WeakStore<Brand, Issuer>}
-   */
-  const brandToIssuer = makeWeakStore('Brand');
-
   let lastLocalIssuerNonce = 0;
```

Comment:
I don't understand the relationship between the `lastLocalIssuerNonce` and the `lastNonce` field of the `LocalDenomState` type, if any.

Reply:
No relationship.  I'll rename `lastNonce` to `lastDenomNonce`, which should help clarify.